frontend/test/model/accessRequests/AccessRequestDisplay.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import { ThemeProvider } from '@mui/material/styles'
+import { render, screen, waitFor } from '@testing-library/react'
+import { UserDisplayProps } from 'src/common/UserDisplay'
+import AccessRequestDisplay from 'src/model/beta/accessRequests/AccessRequestDisplay'
+import { ReviewBannerProps } from 'src/model/beta/reviews/ReviewBanner'
+import { ReviewDisplayProps } from 'src/model/beta/reviews/ReviewDisplay'
+import { lightTheme } from 'src/theme'
+import { testAccessRequest, testAccessRequestReview, testAccessRequestWithComments } from 'utils/test/testModels'
+import { describe, expect, it, vi } from 'vitest'
+
+import { useGetReviewRequestsForModel } from '../../../actions/review'
+
+vi.mock('../../../actions/review', () => ({
+  useGetReviewRequestsForModel: vi.fn(),
+}))
+vi.mock('src/model/beta/reviews/ReviewBanner.tsx', () => ({ default: (_props: ReviewBannerProps) => <></> }))
+vi.mock('src/common/UserDisplay.tsx', () => ({ default: (_props: UserDisplayProps) => <></> }))
+vi.mock('src/model/beta/reviews/ReviewDisplay.tsx', () => ({ default: (_props: ReviewDisplayProps) => <></> }))
+
+describe('AccessRequestDisplay', () => {
+  it('displays the metadata for an access request metadata object when display component is rendered', async () => {
```

Comment:
Repeated yourself here. Maybe `'displays access request metadata when not loading and no errors'`?